src/components/ViewDeck/ViewDeck.js

```diff
@@ -1,94 +1,99 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from "react";
 import { connect } from "react-redux";
-import mapStoreToProps from "../../redux/mapStoreToProps";
+// mapStoreToProps vs. mapStateToProps?
+// import mapStoreToProps from "../../redux/mapStoreToProps";
 import { PieChart } from "react-minimal-pie-chart";
 import "./ViewDeck.css";
 
-class ViewDeck extends Component {
-  state = {
-    hoverCard: "https://i.stack.imgur.com/Vkq2a.png",
-    updatedQty: "",
-    deckQty: "",
-    parsedCards: [],
-  };
-
-  componentDidMount() {
-    this.props.dispatch({
+function ViewDeck({ selectedDeck, user, history, dispatch, cardList }) {
+  //DISPATCHES
+  useEffect(() => {
+    dispatch({
       type: "GET_LIST",
-      payload: this.props.reduxStore.selectedDeck.id,
+      payload: selectedDeck.id,
     });
-
-    this.props.dispatch({
+    dispatch({
       type: "GET_SELECTED_DECK",
-      payload: this.props.history.location.pathname.split("/")[2],
+      payload: history.location.pathname.split("/")[2],
     });
-  }
+  }, []);
 
-  editDeck = (deck) => {
-    console.log("in edit deck with:", deck);
-    const id = this.props.history.location.pathname.split("/")[2];
-    this.props.history.push(`/editdeck/${id}`);
+  //MANA CALCULATIONS
+  useEffect(() => {
+    const includedCards = cardList.filter(
+      (card) => card.deckid === selectedDeck.id
+    );
+    getManaCalculations(includedCards);
+  }, [cardList]);
+
+  const [includedCards, setIncludedCards] = useState([]);
+  const [cmc, setCmc] = useState([]);
+  const [averageCmc, setAverageCmc] = useState(0);
+  const [quantity, setQuantity] = useState(0);
+  const [featuredUri, setFeaturedUri] = useState("");
+  const [hoverCard, setHoverCard] = useState(
+    "https://i.stack.imgur.com/Vkq2a.png"
+  );
+  const [devotion, setDevotion] = useState({
+    White: 0,
+    Blue: 0,
+    Black: 0,
+    Red: 0,
+    Green: 0,
+    Gray: 0,
```

Comment:
these should really be lowercase.

Reply:
whoops!  not sure why I initially made them uppercase.  I just changed them all for the next pr.